assembly-ide/src/assembly/bin/che.bat

```diff
@@ -11,60 +11,19 @@
 
 @echo off
 
-REM Check docker-machine env variables have been added.
-FOR /f "tokens=*" %%j IN ('docker-machine env --shell cmd default') DO %%j
+REM Check to ensure bash is installed
+CALL bash --help > nul 2>&1
+IF %ERRORLEVEL% NEQ 0 goto setup
 
-REM Test to see if docker client is working and reaches docker-machine.
-call docker ps 2> nul
-IF ERRORLEVEL 1 GOTO docker
+REM Launch Che and any associated docker machines, if necessary
+CALL bash --login -i "%~dp0\che.sh" %1 %2 %3 %4 %5 %6 %7 %8
```

Comment:
Added.

I had followed the convention that catalina.bat had been using. This is better syntax.